src/core/analytics.js

```diff
@@ -298,6 +298,11 @@ class Analytics {
       // Load all the client integrations dynamically
       this.clientIntegrations.forEach((intg) => {
         const modName = configToIntNames[intg.name]; // script URL can be constructed from this
+        // Do not continue loading integration that doesn't have mapping
+        if (!modName) {
+          logger.error(`[Analytics] Integration:: ${intg.name} not available for initialization`);
+          return;
+        }
```

Comment:
We should filter these integrations before this step. Otherwise, `clientIntegrations` doesn't reflect the list of valid destinations we expect to successfully load.

Reply:
Done